src/coreclr/nativeaot/Runtime/thread.cpp

```diff
@@ -604,6 +604,7 @@ void Thread::Hijack()
     PalHijack(m_hPalThread, this);
 }
 
+#if !defined(HOST_WASM) // Wasm cannot get the instruction pointer
```

Comment:
```suggestion
#if !defined(HOST_WASM)
```
Hijacking in general does not make sense on WASM.

<details>
<summary>What is "hijacking" in this context?</summary>

On native architectures, it is possible to suspend running managed code by overwriting the return address stored on the stack with a pointer to the EE's "wait for GC" routine - "hijacking" the thread.
</details>

Reply:
Removed.